src/app/core/route-guards/is-maintenance-mode-active.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
```

Comment:
`is` not needed

Reply:
Done

Reply:
It is needed since it is actually an Injectable!!